cluster-autoscaler/cloudprovider/clusterapi/clusterapi_utils.go

```diff
@@ -187,7 +194,14 @@ func parseCPUCapacity(annotations map[string]string) (resource.Quantity, error)
 func parseMemoryCapacity(annotations map[string]string) (resource.Quantity, error) {
 	// the value for the memoryKey is expected to have the unit type included,
 	// eg "1024Mi". if only a number is present, we add the suffix "Mi".
-	val, exists := annotations[memoryKey]
+	var val string
+	var exists bool
+	val, exists = annotations[memoryKey]
```

Comment:
Same as above, this can be done like this
```suggestion
	val, exists := annotations[memoryKey]
```